src/main/java/net/glowstone/block/ItemTable.java

```diff
@@ -301,6 +299,31 @@ private void registerBuiltins() {
         reg(Material.HOPPER_MINECART, new ItemMinecart(GlowMinecart.MinecartType.HOPPER));
         reg(Material.POWERED_MINECART, new ItemMinecart(GlowMinecart.MinecartType.FURNACE));
         reg(Material.STORAGE_MINECART, new ItemMinecart(GlowMinecart.MinecartType.CHEST));
+        
+        //Wearable Items
+        reg(Material.LEATHER_BOOTS, new ItemType().wear(1, ItemWearablePosition.FEET, true));
```

Comment:
Ah, I see it's for determining the armor type of the material? You should use `EnchantmentTarget` for that.